roles/foreman_installer/tasks/module_pr.yml

```diff
@@ -32,3 +32,13 @@
   with_items: "{{ foreman_installer_module_prs.split(',') }}"
   tags:
     - packages
+
+- name: 'Trust github'
```

Comment:
I couldn't really decide where this should go. Maybe a new task (`pre_install`) rather than here?

Reply:
A new task would be good since even if you don't checkout a module PR then you still want this for the devel git checkouts.